src/components/NavBar/NavBarSkipLink.jsx

```diff
@@ -0,0 +1,63 @@
+// React Imports
+import React, { useState } from 'react';
+
+// MUI theme
+import theme from '../../theme';
+
+/**
+ * NavBarSkipLink component is an accessibility component
+ * that allows users to skip to the main content of the page.
+ *
+ * @returns {React.Element} The rendered NavBarSkipLink component.
+ */
+const NavBarSkipLink = () => {
+  // state used for hiding and showing component
+  const [animateOut, setAnimateOut] = useState(true);
+
+  // when TAB key is pressed and bring link into focus
+  const handleFocus = () => {
+    setAnimateOut(false);
+  };
+
+  // handles blur on link
+  const handleBlur = () => {
+    setAnimateOut(true);
+  };
+
+  // handles clicking on link and scrolling to content
+  const handleClick = (event) => {
+    event.preventDefault();
+    setAnimateOut(true);
+    const targetElement = document.getElementById('main-content');
+    if (targetElement) {
+      targetElement.scrollIntoView({ behavior: 'smooth' });
+    }
+  };
+
+  return (
+    <a
+      href="#main-content"
+      onClick={handleClick}
+      onFocus={handleFocus}
+      onBlur={handleBlur}
+      tabIndex={0}
```

Comment:
This is set by default, as it is an interactive element, and is unnecessary so can be removed. See interactive elements: https://html.spec.whatwg.org/multipage/dom.html#interactive-content-2